FerramAerospaceResearch/FARGUI/FAREditorGUI/Simulation/StabilityDerivCalculator.cs

```diff
@@ -59,9 +59,33 @@ public StabilityDerivCalculator(InstantConditionSim instantConditionSim)
             _instantCondition = instantConditionSim;
         }
 
-        public StabilityDerivOutput CalculateStabilityDerivs(double u0, double q, double machNumber, double alpha, double beta, double phi, int flapSetting, bool spoilers, CelestialBody body, double alt)
+        public StabilityDerivExportOutput CalculateStabilityDerivs(CelestialBody body, double alt, double machNumber, int flapSetting, bool spoilers)
```

Comment:
https://github.com/dkavolis/Ferram-Aerospace-Research/blob/f60bec5ce5355f67e1ebedd6ee60a32cd1502bd5/FerramAerospaceResearch/FARGUI/FAREditorGUI/Simulation/StabilityDerivCalculator.cs#L62-L76

Since this is an overload, the behaviour should be the same, instead move the `pressure > 0` check to the implementation.